.env.example

```diff
@@ -29,4 +29,15 @@ OPTIMIZE_CLIENT_SECRET=fill-this-in
 OPTIMIZE_BASE_URL=fill-this-in
 
 ## This value will only change if you're not using SaaS.
-OPTIMIZE_AUDIENCE=optimize.camunda.io
\ No newline at end of file
+OPTIMIZE_AUDIENCE=optimize.camunda.io
+
+## Zeebe API tutorial
+
+## Capture these values from an API client created at the cluster level, with access to the Zeebe scope.
+ZEEBE_CLIENT_ID=fill-this-in
+ZEEBE_CLIENT_SECRET=fill-this-in
+ZEEBE_AUDIENCE=zeebe.camunda.io
+ZEEBE_BASE_URL=fill-this-in
+USER_TASK_KEY=fill-this-in
```

Comment:
Similar comment to the other API tutorial, although I realize this one should reflect what the user actually sees in the UI! Just keep the pattern going for consistence.

Reply:
Updated 👍